src/main/java/mops/Account.java

```diff
@@ -0,0 +1,33 @@
+package mops;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NonNull;
+import lombok.RequiredArgsConstructor;
+
+import java.util.Set;
+
+@Getter
```

Comment:
```suggestion
@Data
```
so that there will be a setter for `image`